modules/monitoring-modifying-core-platform-alerting-rules.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+:_content-type: PROCEDURE
+[id="modifying-core-platform-alerting-rules_{context}"]
+= Modifying core platform alerting rules
+
+As a cluster administrator, you can modify core platform alerting rules to suit your needs. For example, you can tweak an alert expression or change the severity label of an alert.
```

Comment:
```suggestion
As a cluster administrator, you can modify core platform alerts before they are routed by Alertmanager to suit your needs. For example, you can change the severity label of an alert, add a custom label or exclude and alert from being send to Alertmanager.
```
The AlertRelabelConfig can not change the alert expressions as this is done after an alert already fires. Hope these changes clarify that.

Reply:
Revised these sentences using your suggestion as the basis: 

```
As a cluster administrator, you can modify core platform alerts before Alertmanager routes them to a receiver. 
For example, you can change the severity label of an alert, add a custom label, or exclude an alert from being sent to Alertmanager.
```